problem in development\scripts\python\utils\project_utils\generate_documentation.py - ModuleNotFoundError: No module named 'script_database'
